### ISSUE TITLE
🚀 Feature: External Redis Database Secure Connection

### ISSUE THREAD

**Issue by germanllop** [appwrite/appwrite#6869]
### 🔖 Feature description

Enable the option to use a secure connection to an external Redis Database, using -tls or rediss:// to connect to the database.
Right now is only possible to use a non-secure, as the codebase concatenates the connection info with redis:// instead of rediss:// wich is secure.

### 🎤 Pitch

Right now, based on the env vars and the code, appwrite only supports non-secure connections to Redis, so in order to connect to an external Redis Database the only way is to use a non-secure conection.

If we want to scale appwrite and we need to be able to use a centralized databse wich we can scale.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by iamishaan** [comment]
The issue seems to contradict the behaviour and from the description, it looks like features will be affected if the changes are made. Can you elaborate?

**Comment by AudreyLin** [comment]
I would like to work on this. Can you assign it to me?

**Comment by germanllop** [comment]
> The issue seems to contradict the behaviour and from the description, it looks like features will be affected if the changes are made. Can you elaborate?

Yes of course, right now redis connections are not secure between appwrite containers and the redis container, that's on the appwrite repo, it's using redis:// hardcoded to generate the connection string, however, if the redis service goes to an external redis server (Digital Oceaan's Redis Managed Database) by design they use the redis secure protocol, that one start with rediss:// (not redis://) so the only connection that the appwrite containers can made is an unsecure connection.

I can suggest to add an ENV variable to enable secure (rediss://) connections.

**Comment by JoshLee0915** [comment]
I know this is a bit of an older issue, but can I get this assigned to me? I actually have a similar use case atm and I am using a local patch to support secure Redis connections to DigitalOcean that I think I can modify to add this behavior in for Appwrite proper.
